### PR TITLE
add style to organization plugin

### PR DESCRIPTION
**Pull request by 2018D**
## Purpose
Add style sheet to organization plugin.
Minor fix to course_glimpse to retrieve main organization after course refactoring.
Allow only organization_plugin in organization course-detail section.

## Proposal
- [x] add style sheet to organization plugin.
- [x] minor fix to course_glimpse to retrieve main organization.
- [x] allow only organization_plugin in organization course-detail section.

### CODE REVIEW

**Comment by mbenadda** [comment]
Hey David,

I'm not comfortable reviewing/merging the course plugin part without Samuel's input. However, I think we can go forward on the organization plugin styling (maybe with a handful of comments 😇)

Would you mind splitting this and opening a separate PR for the organization plugin?

**Comment by 2018D** [comment]
Fails on the new test: lint-prettier.
What do i do to fix it?

**Comment by mbenadda** [comment]
As of #402 (merged earlier today), we're expecting style files to be formatted through `prettier`, as JS/TS/JSON files already were (and as are python files with `black`).

---

You just have to format this file with `prettier`: `src/richie/apps/courses/templates/courses/plugins/_organization_plugin.scss`.

Personally I find the easiest is to have prettier integrated with my code editor (eg. this extension for Visual Studio Code https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode). I just run it using editor commands / shortcuts / format-on-save.

Otherwise you can run it as a CLI with something like:
```
docker-compose run --rm node yarn prettier --write "src/**/*.+(css|scss)"
```
Which will spin up the node container and have yarn format all the project's css & sass files on disk.

File: src/richie/apps/courses/templates/courses/plugins/_organization_plugin.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
+$richie-organization-plugin-organizations-margin-bottom: $grid-gutter-width;
```
Comment (mbenadda) [inline src/richie/apps/courses/templates/courses/plugins/_organization_plugin.scss]: We need the `!default` flag here to make sure the values are overridable (http://sass-lang.com/documentation/file.SASS_REFERENCE.html#variable_defaults_default)


Also, I'm not sure why you repeat "organizations" in the variable names? Perhaps `$richie-organization-plugin-margin-bottom` is enough? And maybe `$richie-organization-plugin-title-fontsize` etc. for variables related to the title, like you did for the height?

File: src/richie/apps/courses/templates/courses/cms/fragment_course_glimpse.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
 <div class="course-glimpse__content">
     <div class="course-glimpse__content__wrapper">
         <p class="course-glimpse__content__title">{{ course.extended_object.get_title }}</p>
-        <p>{{ course.organization_main.extended_object.get_title }}</p>
+        <p>{{ course.get_main_organization.extended_object.get_title }}</p>
```
Comment (mbenadda) [inline src/richie/apps/courses/templates/courses/cms/fragment_course_glimpse.html]: 👍

File: src/richie/apps/courses/templates/courses/plugins/_organization_plugin.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
+$richie-organization-plugin-margin-bottom: $grid-gutter-width !default;
```
Comment (mbenadda) [inline src/richie/apps/courses/templates/courses/plugins/_organization_plugin.scss]: Sorry! I meant every one of these variables exists to be overridden and needs `!default` 😉

**Comment by sampaccoud** [comment]
Hi @2018D, can you rebase it?

**Pull request merged**